trulens_eval/tests/unit/test.py

```diff
@@ -42,29 +56,257 @@ def requiredonly_test(testmethodorclass):
 
 
 def module_installed(module: str) -> bool:
+    """Check if a module is installed."""
+
     try:
         __import__(module)
         return True
     except ImportError:
         return False
 
 
+T = TypeVar("T")
+
+
+def hashable_skip(obj: T, skips: Set[str]) -> T:
+    """Return a hashable copy of `obj` with all keys/attributes in `skips`
+    removed. 
+    
+    Sequences are returned as tuples and container types are returned as
+    frozendicts. Floats are retuned as 0.0 to avoid tolerance issues. Note that
+    the returned objects are only used for ordering their originals and are not
+    compared themselves.
+
+    Args:
+        obj: The object to remove keys/attributes from.
+
+        skips: The keys to remove.
+    """
+
+    def recur(obj):
```

Comment:
seems easier to just say `recur = lambda obj: hashable_skip(obj, skips=skips)` to me but I'll leave it to you if you want to change it.

Reply:
I think lambdas are generally not recommended in various style guides if they are not needed.